wpeverest/user-registration
# UR-1691 Dev - Compatibility for Conditional logic in row

### All Submissions:






### Changes proposed in this Pull Request:
 This PR adds compatibility codes for row settings that incorporates conditional logic settings for row and repeater settings for repeater row.

### How to test the changes in this Pull Request:

1.  Install conditional logic and repeater fields addon and check if row settings are properly saved, fetched, and updated.

### Types of changes:




### Other information:




### Changelog entry

> Dev - Compatibility for Conditional logic in row.
